Source/JavaScriptCore/jit/AssemblyHelpers.h

```diff
@@ -1742,9 +1742,9 @@ class AssemblyHelpers : public MacroAssembler {
         ok.link(this);
     }
 
-    JS_EXPORT_PRIVATE void cageWithoutUntagging(Gigacage::Kind, GPRReg storage, bool mayBeNull = true);
+    JS_EXPORT_PRIVATE void cageWithoutUntagging(Gigacage::Kind, GPRReg storage);
```

Comment:
Can drop the "WithoutUntagging" part now.

Reply:
Done.